pkg/apis/security/v1alpha1/types.go

```diff
@@ -157,9 +157,35 @@ type NetworkPolicyPort struct {
 	// The port on the given protocol. This can either be a numerical
 	// or named port on a Pod. If this field is not provided, this
 	// matches all port names and numbers.
-	// TODO: extend it to include Port Range.
 	// +optional
-	Port *intstr.IntOrString `json:"port"`
+	Port *intstr.IntOrString `json:"port,omitempty"`
+	// The portRange on the given protocol. This can either be a
+	// a range of ports with some optional exception
+	// or a numerical or named port on a Pod
```

Comment:
Please add "." to end of line for comments. Please check and update all comments.

Reply:
Thx